Extension/src/background/utils/resources.js

```diff
@@ -34,11 +35,9 @@ export const resources = (function () {
      * Foil ability of web pages to identify extension through its web accessible resources.
      *
      * Inspired by:
-     * https://github.com/gorhill/uBlock/blob/7f999b759fe540e457e297363f55b25d9860dd3e/platform/chromium/vapi-background
```

Comment:
thank you for your contribution
just could you return the URL back, it need only `.js` added in the end